inst/include/later.h

```diff
@@ -96,6 +103,10 @@ inline void later(void (*func)(void*), void* data, double secs) {
   later(func, data, secs, GLOBAL_LOOP);
 }
 
+static void later_fd_version_error(void (*func)(int *, void *), void *data, int num_fds, struct pollfd *fds, double secs, int loop_id) {
+  Rf_error("later_fd called, but installed version of the 'later' package is too old; please upgrade 'later' to 1.4.1 or above");
```

Comment:
```suggestion
  (void) func; (void) data; (void) num_fds; (void) fds; (void) secs; (void) loop_id;
  Rf_error("later_fd called, but installed version of the 'later' package is too old; please upgrade 'later' to 1.4.1 or above");
```

Reply:
Again, I do this in my functions, just to be safe against -Wunused-variable.

Reply:
Guess who grew up on Java, not C 😆